src/Avalonia.Controls/SelectedItemChangedEventArgs.cs

```diff
@@ -0,0 +1,36 @@
+﻿using System;
+using System.Collections.Generic;
+using Avalonia;
+using Avalonia.Interactivity;
+
+namespace Avalonia.Controls
+{
+    /// <summary>
+    /// Provides data for the <see cref="TreeView.SelectedItemChanged"/> event.
+    /// </summary>
+    public class SelectedItemChangedEventArgs : RoutedEventArgs
```

Comment:
We already have `SelectionChangedEventArgs` which is used by `SelectingItemsControl`, it might be better to use that? Also that would allow us to add multiple selection to `TreeView` should we wish to without changing the API.

Reply:
Good point for making multiple selections. My reason for using a sperate EventArgs is that `SelectingItemsControl.SelectedItem` is container control, as well as the AddedItems/RemovedItems of `SelectionChangedEventArgs`, but `TreeView.SelectedItem` is the item that the `TreeViewItem` represents. Perhaps, we should make a change on `TreeView.SelectedItem`?

Reply:
Hmm that shouldn't be the case. `SelectingItemsControl.SelectedItem` should represent the item, not the container control. See [this test ](https://github.com/AvaloniaUI/Avalonia/blob/master/tests/Avalonia.Controls.UnitTests/Primitives/SelectingItemsControlTests.cs#L173) for example.